src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
         {
             foreach (ParameterSymbol param in property.Parameters)
             {
-                if (param.RefKind != RefKind.None)
+                if (param.RefKind == RefKind.Ref || param.RefKind == RefKind.Out)
```

Comment:
Correction: `IsConstModReqIsConsumedInRefCustomModifiersPosition_Code_Indexers_Parameters`*

---
In reply to: [120218968](https://github.com/dotnet/roslyn/pull/19658#discussion_r120218968) [](ancestors = 120218968,119789809,118609191,118061963)